armi/reactor/blocks.py

```diff
@@ -1728,6 +1741,12 @@ def createHomogenizedCopy(self, pinSpatialLocators=False):
 
         .. note: If you make a new block, you must add it to an assembly and a reactor.
 
+        Returns
+        -------
+        b
+            A homogenized block containing a single Hexagon Component that contains an
+            average temperature and the number densities from the original block.
```

Comment:
Since it is a homogenized block, I would think only one number density should be assigned.

Reply:
The homogenized block should have a single Component with a single Material (which it does), but that Material can have a mixture of isotopes in it. In this case, the homogenized Material simply contains all of the isotopics from the different Materials of the original Block.